docs/src/pages/demos/autocomplete/IntegrationReactSelect.tsx

```diff
@@ -229,7 +229,7 @@ const components = {
 
 function IntegrationReactSelect() {
   const classes = useStyles();
-  const theme = useTheme<Theme>();
+  const theme = useTheme();
```

Comment:
Should we change the pattern? A user is likely going to use a custom version of the Theme.

Reply:
Could revert that change, though that means we have to specify which theme is used in all calls to `useTheme`, the simplest solutions to get builds to pass was to set a default.

Reply:
> Could revert that change, though that means we have to specify which theme is used in all calls to useTheme, the simplest solutions to get builds to pass was to set a default.

Yes, I think that we should specify which theme is returned in all calls to `useTheme`. People can use the ThemeProvider anywhere in their app. It's consistent with the makeStyles and withStyles API.

Reply:
cc @eps1lon

Reply:
@oliviertassinari @eps1lon Alright, will revert that one.